test/unit/HoldConfirmation.test.js

```diff
@@ -214,8 +217,60 @@ describe('HoldConfirmation', () => {
       expect(main.find('#start-new-search')).to.have.length(1);
       expect(main.find('#start-new-search').text()).to.equal('Start a new search');
     });
+
+    it('should call the React Router context with a url to the homepage.', () => {
```

Comment:
I think the description here should mention about the action to click the button? Maybe this test should be independent from `'If the patron is logged in and the App receives valid data, <HoldConfirmation>'` as it focuses on the default layout without any action from the user has made.

Reply:
Updated the description. I think it's okay to mix layout and action in the same test suite but not necessarily in the same `it` test.